tests/transforms/test_transforms.py

```diff
@@ -1,88 +1,152 @@
-import unittest
 import warnings
 
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_array_less
+import pytest
+from numpy.testing import assert_almost_equal, assert_array_less, assert_array_equal
 from sklearn.pipeline import make_pipeline
 
 from numalogic.base import StatelessTransformer
-from numalogic.transforms import LogTransformer, StaticPowerTransformer, TanhScaler, FlattenVector
+from numalogic.transforms import (
```

Comment:
Which transformers in the list are currently used? The info will help us prioritize which part to code review now.

Reply:
Just the changed files.